pkg/remote/google/repository/mock_AssetRepository.go

```diff
@@ -449,6 +449,29 @@ func (_m *MockAssetRepository) SearchAllNodeGroups() ([]*asset.Asset, error) {
 	return r0, r1
 }
 
+// SearchAllGlobalForwardingRules provides a mock function with given fields:
+func (_m *MockAssetRepository) SearchAllGlobalForwardingRules() ([]*asset.ResourceSearchResult, error) {
```

Comment:
I think you created this manually. You should use the command `mockery --name AssetRepository --dir pkg/remote/google/repository` to update the mock file according to the updated `asset.go` file.